### PR TITLE
gulp build generates browser manifests

### PR DESCRIPTION
**Pull request by andytuba**
Update each browse manifest and in-code version number with values from package.json

**FYI** `npm install` after pulling -- this adds new dependencies.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -94,10 +124,42 @@ gulp.task('build', function(cb) {
 			gulp.src(paths.src)
 				.pipe(gulp.dest(path.join(getBuildDir(browser), paths.dest)));
 		});
+
+		if (config[browser].manifest) {
+			del(path.join(getBuildDir(browser), config[browser].manifest));
```
Comment (erikdesjardins) [inline gulpfile.js]: Does this need to be deleted? Naively I'd assume that gulp would happily overwrite it.
Also, it's async; it should return a promise.

Comment (andytuba) [inline gulpfile.js]: Gulp dest seemed to be throwing "could not write, file exists" errors.. Oh yeah, I suppose I should wait to add new files into old delete is complete.

Comment (erikdesjardins) [inline gulpfile.js]: That's strange, it seems to work fine for me without it :/
Although I have encountered this before; I think I did something weird to fix it (like appending `./` to the dest path)

Comment (andytuba) [inline gulpfile.js]: Yuck, del accepts a callback instead of returning a promise.

Comment (erikdesjardins) [inline gulpfile.js]: [It should](https://www.npmjs.com/package/del#del-patterns-options), we probably need to update to 2.1.0

**Pull request merged**